recipes-graphics/wayland/weston-init/weston.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Weston Wayland Compositor
+RequiresMountsFor=/run
+Requires=weston.path
+
+[Service]
+User=root
+EnvironmentFile=-/etc/default/weston
+ExecStart=/usr/bin/weston-start -v -e -- $OPTARGS
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
an alternative approach is to use udev to run the service when /dev/dri/card0 is there, using ENV{SYSTEMD_WANTS}+="weston.service"